packages/e2e-test-utils-playwright/tsconfig.json

```diff
@@ -4,7 +4,7 @@
 	"compilerOptions": {
 		"incremental": false,
 		"composite": false,
-		"module": "CommonJS",
+		"module": "Node16",
```

Comment:
This needs to be done to fix an error:
```
error TS5110: Option 'module' must be set to 'Node16' when option 'moduleResolution' is set to 'Node16'.
```

I'm not really sure what exactly this does. The matching `moduleResolution: node16` field was added by @swissspidy in https://github.com/WordPress/gutenberg/pull/52993/files#r1288451752